`Type mismatch: inferred type is Activity? but Activity was expected`

We have integrated the SDK and are facing the following issue when running `react-native run-android`:
```
> Task :react-native-medcheck-sdk:compileDebugKotlin FAILED

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.2/userguide/command_line_interface.html#sec:command_line_warnings
61 actionable tasks: 61 executed
Warning: Mapping new ns http://schemas.android.com/repository/android/common/02 to old ns http://schemas.android.com/repository/android/common/01
Warning: Mapping new ns http://schemas.android.com/repository/android/generic/02 to old ns http://schemas.android.com/repository/android/generic/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/addon2/02 to old ns http://schemas.android.com/sdk/android/repo/addon2/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/repository2/02 to old ns http://schemas.android.com/sdk/android/repo/repository2/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/sys-img2/02 to old ns http://schemas.android.com/sdk/android/repo/sys-img2/01
Warning: Mapping new ns http://schemas.android.com/repository/android/common/02 to old ns http://schemas.android.com/repository/android/common/01
Warning: Mapping new ns http://schemas.android.com/repository/android/generic/02 to old ns http://schemas.android.com/repository/android/generic/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/addon2/02 to old ns http://schemas.android.com/sdk/android/repo/addon2/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/repository2/02 to old ns http://schemas.android.com/sdk/android/repo/repository2/01
Warning: Mapping new ns http://schemas.android.com/sdk/android/repo/sys-img2/02 to old ns http://schemas.android.com/sdk/android/repo/sys-img2/01
Note: /home/samkitjain/Samkit/Projects/React/juvoxa-patient-pwa-native/node_modules/@react-native-async-storage/async-storage/android/src/main/java/com/reactnativecommunity/asyncstorage/AsyncStorageModule.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /home/samkitjain/Samkit/Projects/React/juvoxa-patient-pwa-native/node_modules/@react-native-cookies/cookies/android/src/main/java/com/reactnativecommunity/cookies/CookieManagerModule.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: /home/samkitjain/Samkit/Projects/React/juvoxa-patient-pwa-native/node_modules/@react-native-firebase/messaging/android/src/main/java/io/invertase/firebase/messaging/ReactNativeFirebaseMessagingModule.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
e: /home/samkitjain/Samkit/Projects/React/juvoxa-patient-pwa-native/node_modules/react-native-medcheck-sdk/android/src/main/java/com/reactnativemedchecksdk/MedcheckSdkModule.kt: (169, 47): Type mismatch: inferred type is Activity? but Activity was expected

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-medcheck-sdk:compileDebugKotlin'.
> Compilation error. See log for more details
```

--- a/android/src/main/java/com/reactnativemedchecksdk/MedcheckSdkModule.kt
+++ b/android/src/main/java/com/reactnativemedchecksdk/MedcheckSdkModule.kt
@@ -166,7 +166,7 @@ class MedcheckSdkModule(reactContext: ReactApplicationContext) : ReactContextBas
       when (exception.statusCode) {
         LocationSettingsStatusCodes.RESOLUTION_REQUIRED -> try {
           val resolvable = exception as ResolvableApiException
-          resolvable.startResolutionForResult(currentActivity, REQUEST_CHECK_SETTINGS)
+          currentActivity?.let { resolvable.startResolutionForResult(it, REQUEST_CHECK_SETTINGS) }
         } catch (sendEx: SendIntentException) {
           logData("Failed to show dialog" + sendEx)
         } catch (classCast: ClassCastException) {

